src/components/Generic/BaseAccordion.js

```diff
@@ -14,20 +14,20 @@ import {
 } from '../../redux/selectors';
 import Colors from '../../constants/Colors';
 import ResourceCard from '../ResourceCard/ResourceCard';
-import BaseText from '../Generic/BaseText';
+import BaseText from './BaseText';
 import CountIcon from '../Icons/CountIcon';
 import FocusedIcon from '../Icons/FocusedIcon';
 import MarkedIcon from '../Icons/MarkedIcon';
 import CollectionIcon from '../Icons/CollectionIcon';
 
 const SubTypeAccordion = ({
-  subTypeCount,
+  headerCount,
   resourceIds,
   activeCollectionId,
-  subType,
+  headerLabel,
   fromDetailsPanel,
 }) => {
-  const dataArray = [{ title: subType, content: resourceIds }];
+  const dataArray = [{ title: headerLabel, content: resourceIds }];
 
   const renderHeader = (item, expanded) => {
```

Comment:
AFAICT, native-base docs appear to use Class-based React components for their examples.

`renderHeader` and `renderContent` as Class instance method are possibly more performant.  (or, they just haven't updated their docs for Pure Functional Components?)